src/Workspaces/VisualBasic/Portable/Classification/Worker.vb

```diff
@@ -167,6 +167,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Classification
                              SyntaxKind.ExternalSourceKeyword,
                              SyntaxKind.ExternalChecksumKeyword,
                              SyntaxKind.EnableKeyword,
+                             SyntaxKind.ReferenceKeyword,
```

Comment:
Are we keeping both `#r` and `#reference`?  If so, we should probably test both.